cdap-docs/examples-manual/source/examples/spark-page-rank.rst

```diff
@@ -36,17 +40,18 @@ of the application are tied together by the class ``SparkPageRankApp``:
 
 .. literalinclude:: /../../../cdap-examples/SparkPageRank/src/main/java/co/cask/cdap/examples/sparkpagerank/SparkPageRankApp.java
    :language: java
-   :lines: 40-76
+   :lines: 49-96
```

Comment:
You will (probably) need to update the hash of this example in ~/Source/cdap_3.1/cdap-docs/examples-manual/build.sh, if this included file has changed:

```
  test_an_include c914c4c17b0b85bc8541ef00a36bb062 ../../cdap-examples/SparkPageRank/src/main/java/co/cask/cdap/examples/sparkpagerank/SparkPageRankApp.java
```